plugins/modules/dcnm_vrf.py

```diff
@@ -976,182 +1128,69 @@ def diff_for_create(self, want, have):
         json_to_dict_have = json.loads(have["vrfTemplateConfig"])
 
         vlanId_want = str(json_to_dict_want.get("vrfVlanId", ""))
```

Comment:
Should this line 

`vlanId_want = str(json_to_dict_want.get("vrfVlanId", ""))`

be removed now along with the others you removed that you are comparing `json_to_dict_want` and `json_to_dict_have`  using `self.dict_values_differ()`?

Reply:
We talked about this during review.  Please add a comment indicating why this needs to remain.

Reply:
Done.  Also did more refactoring in this method which was just committed.